setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+all_args=$@
+echo $all_args > /tmp/test_file11
+bucket=hailstone-ami-scan-results
+cd /usr/local/bin
+PATH=${PATH}:/usr/local/bin
+echo "after path setup"
+#sleep
+yum install -y epel-release wget
+export SCAP_TARGET='stig-rhel7-disa'
+export TAILORING_SUFFIX='_basefile'
+wget https://raw.githubusercontent.com/CivicActions/drydock/master/baselines/baseline7/scripts/build-yum
```

Comment:
This script is specific to docker (e.g. habitus stuff) - I think we should just reproduce what we need it to do here.